tinyms/serving/server/server.py

```diff
@@ -166,9 +177,10 @@ class Server:
         >>> server = Server()
     '''
 
-    def __init__(self, host='127.0.0.1', port=5000):
+    def __init__(self, host='127.0.0.1', port=5000, server_path=None):
```

Comment:
If you add a new parameter, please add the related annotation.

Reply:
Comment has been added